### DISCUSSION TITLE
vuepress-theme-plume/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/pengzhanbo.cn#33] [Blog-Comment]
# vuepress-theme-plume/

项目地址：vuepress-theme-plume (https://github.com/pengzhanbo/vuepress-theme-plume) npm version npm 基于 vuepress 2.0 制作的 Blog 主题。 ::: warning 提示 基于 VuePress@v2, 仍处于 Beta 阶段。 这意味着功能和AP...

https://pengzhanbo.cn/vuepress-theme-plume/

**Comment by bjxlight** [comment]
> vuepress dev docs

TypeError: Cannot read properties of undefined (reading 'injectStyles')

按照快速开始，编译出现这个错误？

**Reply by pengzhanbo** [reply]
vuepress 更新了 [2.0.0-beta.62](https://github.com/vuepress/vuepress-next/compare/v2.0.0-beta.61...v2.0.0-beta.62) ，除了它列出的一些 破坏性更新，还有依赖的 `vue` 、 `vite` 等也进行了一些重大版本更新。  
当前主题还未完全跟进这些所有依赖项的更新进度，如果使用了高于 `2.0.0-beta.61` 的版本，以及其他更新后的依赖库版本，可能就会出现类似的报错信息。
目前我正在跟进解决中。

**Comment by LiuRan89** [comment]
支持多语言吧

**Reply by pengzhanbo** [reply]
在 `1.0.0-beta.71` 版本中，已支持 多语言。

**Comment by jackercc** [comment]
主题简单大气
